### DISCUSSION TITLE
move node function

### DISCUSSION THREAD

**Discussion by fool-yf** [bcakmakoglu/vue-flow#1519] [Q&A]
Is there a way to trigger a listener method when moving a node with the mouse and the movement ends?

**Comment by bcakmakoglu** [comment]
There are 3 events: [`onNodeDragStart`](https://vueflow.dev/typedocs/interfaces/FlowEvents.html#nodedragstart), [`onNodeDrag`](https://vueflow.dev/typedocs/interfaces/FlowEvents.html#nodedrag), [`onNodeDragStop`](https://vueflow.dev/typedocs/interfaces/FlowEvents.html#nodedragstop)

**Reply by fool-yf** [reply]
I apologize for my previous description. What I meant to ask is if there is an event listener that can be triggered when moving a node using the arrow keys on the keyboard?  @bcakmakoglu
